packages/backend/src/models/SpaceModel.ts

```diff
@@ -448,8 +425,15 @@ export class SpaceModel {
                 `saved_queries_versions.chart_type`,
                 `${PinnedListTableName}.pinned_list_uuid`,
                 `${PinnedChartTableName}.order`,
-                `${ValidationTableName}.error as validation_error`,
-                `${ValidationTableName}.created_at as validation_created_at`,
+                this.database.raw(`
+                    COALESCE(
+                        (
+                            SELECT json_agg(validations.*) 
+                            FROM validations 
+                            WHERE validations.saved_chart_uuid = saved_queries.saved_query_uuid
+                        ), '[]'
+                    ) as validation_errors
+                `),
```

Comment:
What do we think now? @ZeRego & @rephus 

looking good on the FE, but will make changes to FE after this PR
![image](https://github.com/lightdash/lightdash/assets/7611706/105dbc5c-40fb-4408-9d9c-0df0eb845878)